### PR TITLE
Added `canister_info` management canister API

### LINKED ISSUE #1281: Go over all IC Wasm APIs and cdk-rs APIs and make sure we have everything implemented
- [x] Add the [canister_info](https://docs.rs/ic-cdk/latest/ic_cdk/api/management_canister/main/fn.canister_info.html) API to the management canister. It's been in ic-cdk since v0.9.2.
- [x] Add stableGrow and stableGrow64 to when azle runs in node

### PR DESCRIPTION
**Pull request by dansteren**
Call it like this:

```ts
import { Canister, ic, update } from 'azle';
import {
    CanisterInfoArgs,
    CanisterInfoResult,
    managementCanister
} from 'azle/canisters/management';

export default Canister({
    getCanisterInfo: update(
        [CanisterInfoArgs],
        CanisterInfoResult,
        async (args) => {
            const result = await ic.call(managementCanister.canister_info, {
                args: [
                    {
                        canister_id: args.canister_id,
                        num_requested_changes: args.num_requested_changes
                    }
                ]
            });
            return result;
        }
    )
});
```

Closes #1281

### CODE REVIEW

File: examples/management_canister/src/index.ts [hunk 1]
```diff
@@ -113,6 +115,21 @@ export default Canister({
 
         return true;
     }),
+    getCanisterInfo: update(
+        [CanisterInfoArgs],
+        CanisterInfoResult,
+        async (args) => {
```
Comment (lastmjs) [inline examples/management_canister/src/index.ts:121]: Just pass the args into the cross canister call

File: src/lib_new/ic.ts [hunk 2]
```diff
@@ -929,9 +929,11 @@ export const ic: Ic = globalThis._azleIc
           replyRaw: () => {},
           setCertifiedData: () => {},
           stableBytes: () => {},
+          stableGrow: () => {},
```
Comment (lastmjs) [inline src/lib_new/ic.ts:932]: Why have these been removed?

File: canisters/management/canister_management.ts [hunk 3]
```diff
@@ -9,6 +9,20 @@ import {
     blob
 } from '../../src/lib_functional';
 
+export {
```
Comment (lastmjs) [inline canisters/management/canister_management.ts:12]: What's going on with this structure?

**Review by lastmjs — CHANGES_REQUESTED** [review]

**Review by lastmjs — CHANGES_REQUESTED** [review]

**Comment by lastmjs** [comment]
Also put the license comments

File: examples/management_canister/src/index.ts (hunk 1 above)
Comment (dansteren) [reply examples/management_canister/src/index.ts:121]: Turns out I greatly exaggerated, and canisterStatus was really the only other place this happened...

Both are fixed.

File: canisters/management/canister_management.ts (hunk 3 above)
Comment (dansteren) [reply canisters/management/canister_management.ts:12]: done

**Review by lastmjs — APPROVED** [review]

**Pull request merged**